compute/client_library/snippets/tests/test_ip_address.py

```diff
@@ -268,3 +269,18 @@ def test_release_static_ip(static_ip: Address):
     )
     ips = list_ip_addresses(client, PROJECT, region=region)
     assert static_ip.name not in ips
+
+
+@pytest.mark.parametrize(
+    "static_ip", [{"region": "us-central1"}], indirect=True
+)
+def test_assign_static_ip_to_existing_vm(instance_with_ips: Instance, static_ip: Address):
+    PROJECT = google.auth.default()[1]
+    ZONE = "us-central1-b"
+    REGION = "us-central1"
+
+    client = AddressesClient()
+    ip_address = client.get(project=PROJECT, region=REGION, address=static_ip.name)
+
+    updated_instance = assign_static_ip_to_existing_vm(PROJECT, ZONE, instance_with_ips.name, ip_address.address)
+    assert updated_instance.network_interfaces[0].access_configs[0].nat_i_p == ip_address.address
```

Comment:
Does the machine have to be stopped before modifying the network interface? I honestly don't remember :)

Reply:
No, it can run 👍